src/oterm/app/chat_edit.py

```diff
@@ -23,6 +24,9 @@
 from oterm.tools import available as available_tool_defs
 from oterm.types import Tool
 
+class OtermOllamaOptions(Options):
+    class Config:
+        extra = "forbid"
```

Comment:
I believe this is the only way to force a pydantic model to not allow extra fields when parsing a json string